src/lib/services/licence-gauging-stations-service.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+const licenceGaugingStationsRepo = require('../connectors/repos/licence-gauging-stations');
+
+const createNewLicenceLink = async (gaugingStationId, licenceId, properties = {
+  licenceVersionPurposeConditionId: null,
+  thresholdUnit: 'mAOD',
+  thresholdValue: 0,
+  abstractionPeriod: null,
+  restrictionType: 'flow',
+  alertType: 'reduce',
+  source: 'wrls'
+}) => licenceGaugingStationsRepo.create({ gaugingStationId, licenceId, ...properties, date_status_updated: new Date() });
```

Comment:
I have a vague recollection we were going to set `date_status_updated` when a notification is sent out.

I wonder then if `date_status_updated` should be `null` until then?

Reply:
It's currently not nullable, but if you feel this is a problem we can create a migration script to make it nullable. What do you think?

Reply:
Discussed with James. This will be resolved in commit d6436849